scripts/1_generate_database.sql

```diff
@@ -281,7 +281,7 @@ CREATE TABLE `EventOccurrences` (
     `EventFinish`       DATETIME            NOT NULL        COMMENT 'Use UTC time',
     `Pattern`           TINYINT UNSIGNED    DEFAULT NULL    COMMENT 'Patterns:\n0 - Daily,\n1 - Weekly,\n2 - AbsoluteMonthly,\n3 - RelativeMonthly',
     `Storage`           BIGINT UNSIGNED     NOT NULL,
-
+	`Color`				INT 				NOT NULL,
```

Comment:
I think better to use new script with alter table than  to initialize a new db with different data.